editors/vim/syntax/fluent.vim

```diff
@@ -0,0 +1,26 @@
+if exists("b:current_syntax")
+    finish
+endif
+
+syntax match fluentComment "\v#.*$"
+syntax match fluentIdentifier "^\v[a-zA-Z-]+" nextgroup=fluentDelimiter
```

Comment:
Can you make this a bit more like the actual `Identifier` grammar? In particular, allowing digits in non-first position is important for versioned strings.